src/main/java/chzzk/grassdiary/domain/diary/DiaryImage.java

```diff
@@ -0,0 +1,22 @@
+package chzzk.grassdiary.domain.diary;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.OneToOne;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Getter
+@NoArgsConstructor
+@Entity
+public class DiaryImage {
+    @Id
+    @GeneratedValue( strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @OneToOne
```

Comment:
```suggestion

    @OneToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "diary_id")
```
이하 지연로딩부분은 코드 수정할 때 보기 불편하실 것 같아 코드 리뷰 생략하겠습니다.